### DISCUSSION TITLE
Combobox + Input

### DISCUSSION THREAD

**Discussion by ceyerkes** [twilio-labs/paste#3152] [General] [answered]
hello hello hello - @laurenfb raised a good question about comboboxes + inputs. We'd like to use a combobox for the font size picker in the RTE, but we'd also like to allow people to enter a number not included in the dropdown. It seems as though those two actions are mutually exclusive, so I wanted to check with the team to see if we could work around this. Open to other ideas as well.

![Screenshot 2023-04-12 at 11 04 53 AM](https://user-images.githubusercontent.com/87141551/231530687-bb60a39a-286c-4ca0-b274-dff5d4d4cda1.png)

Thanks!

cc: @laurenfb please chime in if i missed anything!

**Comment by laurenfb** [comment]
Ack, I would not phrase it that way @ceyerkes! More detail - 

We are already using a paste combobox component for the font size and font family. This allows both input and a dropdown. The issue is not the combobox itself.

The issue is that product would like there to be a caret on the right side of the comboboxes that also use autocomplete. I think Paste specifically chooses not to use that caret for the autocomplete/typeable combobox because it performs a different action. Product wants us to see if we can add a caret to the autocomplete combobox.

I can see both sides - not having a caret on all dropdowns may look a little inconsistent to users, but the actions you can take on each type are semantically different, so I can see why Paste does not include the caret.

**Reply by ceyerkes** [reply]
uh oh, sorry about that!

**Comment by SiTaggart** [comment, marked as answer]
Hi Folks

@laurenfb is correct, it's an intentional omission of the caret when it's in autocomplete mode, because 9 times out of 10 that is the most appropriate thing to do. I totally get that it might feel a little odd for this use case.

It's pretty rare this comes up, in fact this use case is the only time it has done, so I doubt we'll do anything about it for the foreseeable future until we see more signal that it's becoming a problem for more products.

I hope that makes sense.
